Race/race.py

```diff
@@ -0,0 +1,75 @@
+import random
+import time
+
+
+class Animal:
+
+    def generatenumber(self):
+        num = random.randint(1, 10)
+        return num
+
+    def move(self):
+        return self.generatenumber()
+
+
+class TortoiseClass(Animal):
+    def move(self,tortoise):
```

Comment:
let's make the position of animal a field in animal base class containing a default value of 1.